packages/bitcore-lib/lib/publickey.js

```diff
@@ -281,6 +296,115 @@ PublicKey.fromX = function(odd, x) {
   });
 };
 
+/**
+ * PublicKey instance from a Taproot (32-byte) public key
+ * @param {String|Buffer} hexBuf 
+ * @returns {PublicKey}
+ */
+PublicKey.fromTaproot = function(hexBuf) {
```

Comment:
Consider refactoring fromTaproot and isValidTaproot to abstract common functionality, which:
- Updates param to Buffer IF param is hex string
- Ensures param is now 32 byte Buffer